### PR TITLE
Update cassandra docs

### PR DESCRIPTION
**Pull request by AdrianRaFo**
Move cassandra docs to frees-cassandra repo and copy them to publish microsite

### CODE REVIEW

**Review by juanpedromoreno — APPROVED** [review]

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 resolvers += Resolver.sonatypeRepo("releases")
 
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0")
```
Comment (juanpedromoreno) [inline project/plugins.sbt]: Why are you including coursier?

Comment (AdrianRaFo) [reply project/plugins.sbt]: I need it to download the tut plugin

Comment (juanpedromoreno) [reply project/plugins.sbt]: It's not necessary, `tut` is already being used in several modules.

**Review by juanpedromoreno — CHANGES_REQUESTED** [review]
Let's remove coursier.

**Review by juanpedromoreno — APPROVED** [review]

**Pull request merged**